.github/workflows/push.yml

```
@@ -17,86 +18,131 @@ on:
+            git push
+            echo "Successfully deployed to production"
+          fi
 
 
-  notify-discord-merge:
-    needs: [setup]
-    name: Notify Discord (merged to main)
+  notify-discord-push:
+    needs: [build-docs, deploy-production]
+    name: Notify Discord (push to main)
     runs-on: ubuntu-20.04
```

actionlint (1.7.8)

label "ubuntu-20.04" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)